packages/server/src/mocha-setup.ts

```diff
@@ -19,15 +19,24 @@ import { Workspace } from './api/workspace/workspace.model';
 import server from './index';
 import { ReportSchema } from './api/report-schema/report-schema.model';
 
+// Set NO_MOCHA_INTEGRATION to true when running test not requiring database or server
```

Comment:
As per our discussion...let's either remove this in the PR and just treat all tests as "integration" tests for now or address the unit and integration test shortfalls now.

Reply:
Would you be OK to wait until this work gets prioritized? https://app.asana.com/0/1188940305683068/1200662026396299/f